dashboard/config/scripts_json/csd7-2021.script_json

```diff
@@ -2994,7 +2994,7 @@
       "key": "aa9a6902-38d1-4613-8535-7b515b0f7e04",
       "position": 16,
       "properties": {
-        "description": "Once students have taken the survey, redirect them to the front of the room to see what happens with all of the data they've been generating.\n\n<i class=\"fa fa-list-alt\" aria-hidden=\"true\"></i> **Video:** Show students the \n[r finding_patterns_in_data/aiml/2021] video.",
+        "description": "Once students have taken the survey, redirect them to the front of the room to see what happens with all of the data they've been generating.\n\n<i class=\"fa fa-list-alt\" aria-hidden=\"true\"></i> **Video:** Show students the \n[r finding_patterns_in_data/csd/2021] video.",
```

Comment:
is there any follow-up needed to prevent this kind of problem from happening again in the future? it seems like an issue with lesson copy, but not sure if it's since been addressed

Reply:
I addressed issues in #45979 and #46114, but I think there might be one more to come (I forgot to add `purpose` to the list of fields to update). Each time, I'm adding more tests but with issues like forgetting to update a field, I'm not sure the best way to ensure they're not forgotten

Reply:
I see. Here is an idea:
1. extract this into a constant array `MARKDOWN_FIELDS`: https://github.com/code-dot-org/code-dot-org/blob/5e0e00bd40653e39771bfea07395f76ea9a717ed/dashboard/app/models/lesson.rb#L829
3. make this method raise if you try to render something that is not in the new `MARKDOWN_FIELDS` array: https://github.com/code-dot-org/code-dot-org/blob/5e0e00bd40653e39771bfea07395f76ea9a717ed/dashboard/app/models/lesson.rb#L175

happy to own follow-up if that feels like too much.